src/bin/pgcopydb/schema.c

```diff
@@ -423,11 +423,20 @@ schema_list_extensions(PGSQL *pgsql, DatabaseCatalog *catalog)
 	SourceExtensionArrayContext parseContext = { { 0 }, catalog, false };
 
 	char *sql =
+		"with recursive fk_constraints as ("
+		"     select * from pg_constraint where contype = 'f' and conrelid <> confrelid"
+		" ), ordered_fk_constraints as ("
+		"     select distinct c.confrelid as relid, 0 depth from fk_constraints c where not exists (select * from fk_constraints fc where fc.conrelid = c.confrelid)"
+		"     UNION"
+		"     select distinct c.conrelid as relid, r.depth + 1 FROM ordered_fk_constraints r join fk_constraints c ON c.confrelid = r.relid"
+		" ), min_constraint AS (select relid, min(depth) as depth FROM ordered_fk_constraints group by relid),"
+		" preprocessed as ("
```

Comment:
Please reformat the SQL query for our line length limit (target < 80 for readability, allow oversize lines when you feel like it's better, up to 90 chars).

Reply:
Done